xml/System/TypeInitializationException.xml

```diff
@@ -58,68 +58,68 @@
   
- The following example shows a <xref:System.TypeInitializationException> exception thrown by a compiler-generated static constructor. The `Example` class includes a `static` (in C#) or `Shared` (in Visual Basic) field of type `TestClass` that is instantiated by passing a value of 3 to its class constructor.  That value, however, is illegal; only values of 0 or 1 are permitted. As a result, the `TestClass` class constructor throws an <xref:System.ArgumentOutOfRangeException>. Since this exception is not handled, it is wrapped in a <xref:System.TypeInitializationException> exception.  
+      Note that the exception message displays information about the <xref:System.Exception.InnerException%2A> property.  
   
- [!code-csharp[System.TypeInitializationException#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.TypeInitializationException/cs/ctorException1.cs#3)]
- [!code-vb[System.TypeInitializationException#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.TypeInitializationException/vb/CtorException1.vb#3)]  
+- A missing assembly or data file  
+ 
+   A common cause of a <xref:System.TypeInitializationException> exception is that an assembly or data file that was present in an application's development and test environments is missing from its runtime environment. For example, you can compile the following example to an assembly named Missing1a.dll by using this command-line syntax:  
   
- Note that the exception message displays information about the <xref:System.Exception.InnerException%2A> property.  
+   ```csharp  
+   csc -t:library Missing1a.cs  
+   ```  
   
- A missing assembly or data file  
- A common cause of a <xref:System.TypeInitializationException> exception is that an assembly or data file that was present in an application's development and test environments is missing from its runtime environment. For example, you can compile the following example to an assembly named Missing1a.dll by using the this command-line syntax:  
+   ```vb  
+   vbc Missing1a.vb -t:library  
+   ```  
   
-```csharp  
-csc /t:library Missing1a.cs  
-```  
+   [!code-csharp[System.TypeInitializationException#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.TypeInitializationException/cs/Missing1a.cs#1)]
+   [!code-vb[System.TypeInitializationException#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.TypeInitializationException/vb/Missing1a.vb#1)]  
   
-```vb  
-vbc Missing1a.vb /t:library  
-```  
+   You can then compile the following example to an executable named Missing1.exe by including a reference to Missing1a.dll:  
   
- [!code-csharp[System.TypeInitializationException#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.TypeInitializationException/cs/Missing1a.cs#1)]
- [!code-vb[System.TypeInitializationException#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.TypeInitializationException/vb/Missing1a.vb#1)]  
+   ```console
+   csc Missing1.cs /r:Missing1a.dll  
```

Comment:
should you change to - here too? the downside of changing to console is that this would show up even if you have  VB selected for this API. same for the next snippet.